deploy/bin/github-actions-plan-deployment

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo "Found GitHub ref name '$GITHUB_REF_NAME', repository '$GITHUB_REPOSITORY'"
+
+STACK=""
```

Comment:
make STACK a flag instead of having your example in here?

Reply:
I've changed it so that you make a branch of the format deploy/stackname, and it deploys the stack that matches the name of the branch. I've updated README.md.